README.md

```diff
@@ -16,3 +16,29 @@ To build the project run:
 cd ledger-core/v2
 make vendor pre-build build
 ````
+
+
+## Prometheus
+
+Prometheus can be deployed via `make kube_apply_prometheus` command.
+
+Prometheus is scraping all targets that have prometheus annotations:
+```
+      annotations:
+        prometheus.io/port: "8001"
+        prometheus.io/scrape: "true"
+```
+
+You can specify the port which your app is using. Any new deployed software with correct annotations will be scraped by prometheus.
+
+After prometheus and ingress controller are up and running, you can check what's going on:
+
+To get a list of metrics names and their series count:
+```
+curl -s 'http://localhost/prometheus/api/v1/status/tsdb' | jq .
```

Comment:
`localhost:8001` I think

Reply:
Nope, it's about access to prometheus itself, not metrics endpoint of any part of platform. And it's defined as Ingress object here ledger-core/v2/scripts/deploy/kube/prometheus/05-prometheus-Ingress.yaml